sktime/performance_metrics/forecasting/_classes.py

```diff
@@ -309,11 +397,13 @@ def make_forecasting_scorer(func, name=None, greater_is_better=False):
     scorer:
         Metric class that can be used as forecasting scorer.
     """
-
-    return _MetricFunctionWrapper(func, name=name, greater_is_better=greater_is_better)
+    if greater_is_better:
+        return _BaseForecastingErrorMetric(func, name=name, multioutput=multioutput)
```

Comment:
Just read in scikit-learn that 
> All scorer objects follow the convention that higher return values are better than lower return values.

https://scikit-learn.org/stable/modules/model_evaluation.html#common-cases-predefined-values

Not sure if we also want to adopt that convention or give users the flexibility. I'd say we leave it as it is for now. Having a single convention would be simpler for higher-level functionality though.

Reply:
I agree. I was looking at that at the end of the original PR (that's why I briefly had the code added to return the negative of a metric if `greater_is_better=True`). I believe scikit-learn does something like that. 

I think you mentioned that all of that is currently handled when the metrics are called.  I think in the long-run it makes sense to move that into the metrics? They way I have it written just the call in `_BaseForecastingSuccessMetric()` would need the negation logic. I'd be down to hold that to follow-on PR (we don't have any metrics where `greater_is_better=True` currently).

Reply:
Yes okay let's do that in a separate PR when necessary, currently it's handled when the metrics are called/compared.